### PR TITLE
Make TxnFees back to void Coinbase address

### PR DESCRIPTION
**Pull request by kaikawaliu**
## Backward Compatibility

- [ ] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: constants_local.xml [hunk 1]
```diff
@@ -309,7 +309,7 @@
         <TXN_PERSISTENCE_NAME>txnsbackup</TXN_PERSISTENCE_NAME>
         <ENABLE_TXNS_BACKUP>false</ENABLE_TXNS_BACKUP>
         <SHARDLDR_SAVE_TXN_LOCALLY>false</SHARDLDR_SAVE_TXN_LOCALLY>
-        <BLOOM_FILTER_FALSE_RATE>0.000001</BLOOM_FILTER_FALSE_RATE>
+	    <BLOOM_FILTER_FALSE_RATE>0.000001</BLOOM_FILTER_FALSE_RATE>
```
Comment (KaustubhShamshery) [inline constants_local.xml:312]: adjust the indent

Comment (ansnunez) [reply constants_local.xml:312]: Looks like it's not necessary to change this file. Please remove from the PR

File: src/libNode/MicroBlockPreProcessing.cpp [hunk 2]
```diff
@@ -1339,12 +1336,10 @@ bool Node::CheckMicroBlockHashes(bytes& errorMsg) {
...
       m_mediator.m_ds->m_mode != DirectoryService::IDLE) {
-    // Check COINBASE_REWARD_PER_DS + totalTxnFees
-    uint128_t rewards = 0;
-    if (!SafeMath<uint128_t>::add(m_mediator.m_ds->m_totalTxnFees,
-                                  COINBASE_REWARD_PER_DS, rewards)) {
-      LOG_GENERAL(WARNING, "total_reward addition unsafe!");
-    }
+    // Check COINBASE_REWARD_PER_DS
+
+    uint128_t rewards = COINBASE_REWARD_PER_DS;
```
Comment (ansnunez) [inline src/libNode/MicroBlockPreProcessing.cpp:1341]: Can just use `COINBASE_REWARD_PER_DS` directly

**Review by ansnunez — CHANGES_REQUESTED** [review]

File: src/libNode/MicroBlockPreProcessing.cpp (hunk 2 above)
Comment (kaikawaliu) [reply src/libNode/MicroBlockPreProcessing.cpp:1341]: I'm worrying we might have to adjust this number, so easier to maintain if define exceptionally

**Review by KaustubhShamshery — APPROVED** [review]

**Review by ansnunez — APPROVED** [review]

**Pull request merged**